packages/block-editor/src/components/warning/index.js

```diff
@@ -16,40 +16,39 @@ function Warning( { className, actions, children, secondaryActions } ) {
 			<div className="block-editor-warning__contents">
 				<p className="block-editor-warning__message">{ children }</p>
 
-				{ Children.count( actions ) > 0 && (
-					<div className="block-editor-warning__actions">
-						{ Children.map( actions, ( action, i ) => (
+				<div className="block-editor-warning__actions">
+					{ Children.count( actions ) > 0 && (
```

Comment:
For what reason did you invert the condition and the div here. It seems logical for me that the div is not shown if there's no action.

Reply:
Yes, so I think I did this because we had two divs of action buttons and meant to visually align them next to each other to appear as a single row of buttons. By having them in the same div it enables cleaner code. But I'm up for alternative fixes!